UTILITIES/Calc_Delta.py

```diff
@@ -0,0 +1,67 @@
+###############################################################################
```

Comment:
I think that this could be shorter:

e.g.:

```python
from astropy.time import Time
t1 = Time(start)
t2 = Time(stop)
dt = t2-t1
delta_secs = dt.sec
delta_minutes = delta_secs/60.0
delta_hours = delta_secs/3600.0
```

Reply:
Your scheme is smaller but I want Calc_Delta to be able to handle both Chandra DOY strings and Chandra seconds as inputs. Your scheme cannot handle Chandra seconds as the format is ambiguous.